bam/src/configuration/applier/kpi.cc

```diff
@@ -141,9 +141,11 @@ void applier::kpi::apply(
   // Delete objects.
   for (std::map<unsigned int, applied>::iterator
          it(to_delete.begin()),
+         next_it(to_delete.begin()),
```

Comment:
IMHO _next_it_ is unnecessary as the _to_delete_ container itself is not modified (and _it_ iterator is not invalidated).